packages/examples/.storybook/preview.js

```diff
@@ -1,6 +1,8 @@
-import { addParameters } from "@storybook/react";
+import { withDesign } from "@storybook/addon-designs";
 
-addParameters({
+export const parameters = {
   viewMode: "story",
   docs: { page: null },
-});
+};
+
+export const decorators = [withDesign];
```

Comment:
This addon does not require `withDesign` decorator, see #144 . It has been there for unknown reasons...

```suggestion
```

We should remove this decorator in this major release (not necessary in this PR).
related: https://github.com/storybookjs/addon-designs/pull/192/files#r1225644956

Reply:
Went ahead and resolved this comment by removing in this PR.